app/clients/sms/aws_sns.py

```diff
@@ -58,6 +58,15 @@ def send_sms(self, to, content, reference, multi=True, sender=None):
                     'StringValue': sender
                 }
 
+            # If the number is US based, we must use a US Toll Free number to send the message
+            country = phonenumbers.region_code_for_number(phonenumbers.parse(to))
```

Comment:
Are you able to reuse `to` or `match` or another attribute to avoid parsing twice the phone number?

Reply:
It appears that `region_code_for_number` requires a `numobj` which can be acquired by parsing a string.

From my understanding the way to get there is parsing `to` which is a string acquired by parsing `match`.

The `match.number.national_number` returns an `int` which is not accepted by `region_code_for_number`

Because of this, I believe we must go from `match.number -> to -> region_code_for_number(to)` for the method to work as intended. But it's also possible that I have missed something!

Reply:
Changed parameter sent to `region_code_for_number` to `match.number` as it fulfills the `numobj` type requirement